Assets/Scripts/HandAnimatorController.cs

```diff
@@ -28,11 +28,34 @@ private void Awake()
     private void Update()
     {
         if (FirstPersonController.Instance.currentInteractable &&
-            FirstPersonController.Instance.currentInteractable.GetType() == typeof(Socket)) return;
+            FirstPersonController.Instance.currentInteractable.GetType() == typeof(Socket))
+        {
+            if (Mouse.current.leftButton.isPressed)
+            {
+                _animController.SetBool("extendArms", false);
+            } 
             
+            return;
```

Comment:
Is it necessary to write a return here? Let me know if I'm wrong, I'm just curious

Reply:
It is not "Necessary" but since the rest of the function is useless if the first part applies, it's performance efficient to just skip it with what's called an "Early Return"